scout/server/blueprints/cases/controllers.py

```diff
@@ -605,6 +607,10 @@ def case_report_content(store, institute_id, case_name):
             **DISMISS_VARIANT_OPTIONS,
             **CANCER_SPECIFIC_VARIANT_DISMISS_OPTIONS,
         }
+        data["genes_in_panels"] = store.panels_to_genes(
```

Comment:
This step might take quite some time according to the size of the SVs, that's why I'm including it in the condition `if data["cancer"]:`. I'm open to suggestions here.